apps/tx-builder/src/scripts/deploy-test-basic-contract.js

```diff
@@ -0,0 +1,36 @@
+const RINKEBY_ETHERSCAN_URL = 'https://rinkeby.etherscan.io'
```

Comment:
would be great to have it as an environment variable in case we want to deploy it on another chain

Reply:
good catch! I will update it!